student

```diff
@@ -0,0 +1,20 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom, :rentals
+
+  def initialize(classroom:, age:, parent_permission: true, name: 'Unknown')
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+    @rentals = []
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
+end
```

Comment:
- [ ] I don't know if you missed this file, but please consider deleting it since it has no extension and you already have a `student.rb` file in your project. Remember that having duplicate files can make it hard for people reading your code to collaborate and understand your project! 🤓 🔧

Reply:
Hi @anagudelogu thanks for noticing it.  I wasn't aware of it.  I deleted it.